src/ert/parsing/lark_parser_error_info.py

```diff
@@ -0,0 +1,74 @@
+from dataclasses import InitVar, dataclass
+from typing import List, Optional
+
+from typing_extensions import Self
```

Comment:
This file should be named `error_info.py` . The prefix is superflous. Same with `file_context_token.py` and `primitive_token.py`

Reply:
Done